extensions/origins/source/protocols/pubsub/index.ts

```diff
@@ -0,0 +1,8 @@
+import { create } from './Aspect'
+import type { Protocol } from '../index'
+
+export = {
+  id: 'pubsub',
+  protocols: ['pubsub:'],
```

Comment:
It seems like we once again on the way to invent URIs that nobody else is using. And I believe this way proven to be problematic on initial storage incarnation.

Reply:
This is not something added in this PR.

Changing this will require refactoring of all existent protocols (http and amqp). I'd like to do that actually, but not today.